src/sourmash/commands.py

```diff
@@ -820,6 +860,16 @@ def multigather(args):
 
             # downsample if requested
             if args.scaled:
```

Comment:
When the `args.scaled` value is zero, this if condition will never be executed. That is the reason it's not existing.

Reply:
Oh I see, is that the whole issue here?

Reply:
Yes! And now I'm getting the error message "ERROR: --scaled value must be >= 1".

Reply:
Oh okay. So that's the expected error message, and the test should pass now?

Reply:
Yes, it should pass now. And I think you should insert your `if args.scaled == 0` at the very beginning of the function to avoid doing any processing before checking the value of scaled.

Reply:
Alright! Is that the only change?

Reply:
I think so.. Let's re-check after doing it.

Reply:
Okay